Projects/Contracts/Sources/View/CrossSellingItem.swift

```diff
@@ -51,6 +51,7 @@ struct CrossSellingItem: View {
                     hText(L10n.crossSellGetPrice)
                 }
                 .hButtonConfigurationType(.primaryAlt)
+                .frame(width: 106)
```

Comment:
Can we skip this setting of fixed width?

Reply:
how?